manifests/python.yml

```diff
@@ -639,6 +639,10 @@ tests/:
         flask-poc: v2.11.0
         uds-flask: v2.11.0
         uwsgi-poc: v2.11.0
+    test_debugger_inproduct_enablement.py:
```

Comment:
Can we enable for Python or will that be in a follow-up?

Reply:
The last time I've checked the in-product enablement was still not merged to master. Since I don't have expected version to put here, it marked as missing feature.